### PR TITLE
format coordinates sample

### PR DESCRIPTION
**Pull request by tschie**
Shows how to convert coordinates to strings in different formats

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 Please review when you get a chance.

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ notations including decimal degrees; degrees, minutes, seconds; Universal Transverse Mercator (UTM), and United 
+ States National Grid (USNG).</p>
+
+<h2>How to use the sample</h2>
+
+<p>Click on the map to see a callout with the clicked location's coordinate formatted in 4 different ways. You can 
+also put a coordinate string in any of these formats in the text field. Hit Enter and the coordinate string will be 
+converted to a map location which the callout will move to.</p>
+
+<p><img src="FormatCoordinates.png"/></p>
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md]: In the wrong place should be right before `<h2>How to use the sample</h2>`.

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/FormatCoordinatesSample.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      input.setOnAction(e -> {
+        String inputText = input.getText();
+        if (!"".equals(inputText)) {
+          // try each coordinate format converter, use the first one with the correct format
+          try {
+            Point point = CoordinateFormatter.fromLatitudeLongitude(inputText, map.getSpatialReference());
+            showCalloutWithLocationCoordinates(point);
+            return;
+          } catch (ArcGISRuntimeException ex) {
+            //ignore
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/FormatCoordinatesSample.java]: Would put `ignore, wrong format`.  Easier to read.

**Comment by john0005** [comment]
Might be a Mac thing but callout not showing all of the `Degrees, Minutes, Seconds`.
screen shot 2018-02-28 at 1 00 50 pm

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<h1>Format Coordinates</h1>
+
+<p>Demonstrates how to convert between map location Point in WGS84 and a String in a number of different coordinate
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md]: `Demonstrates how to convert between a map location Point in WGS84 or a coordinate String to a number of different ..`

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+<p>Click on the map to see a callout with the clicked location's coordinate formatted in 4 different ways. You can 
+also put a coordinate string in any of these formats in the text field. Hit Enter and the coordinate string will be 
+converted to a map location which the callout will move to.</p>
+
+<p><img src="FormatCoordinates.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To convert between <code>Point</code>s and different coordinate formats using the
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md]: `between a <code>Point</code>` no `s`.

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+<p><img src="FormatCoordinates.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To convert between <code>Point</code>s and different coordinate formats using the 
+<code>CoordinateFormatter</code>:</p>
+<ol>
+    <li>Get or create a map <code>Point</code> with a spatial reference.</li>
+    <li>Use one of the static methods on <code>CoordinateFormatter</code> such as
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/README.md]: Would say `Use one of the static "to" methods`

File: src/main/java/com/esri/samples/displayinformation/format_coordinates/FormatCoordinatesSample.java [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+            "USNG: " + usng + "\n"
+    );
+    mapView.getCallout().showCalloutAt(location, new Duration(500));
+  }
+
+  /**
+   * Stops and releases all resources used in application.
+   */
+  @Override
+  public void stop() throws Exception {
```
Comment (john0005) [inline src/main/java/com/esri/samples/displayinformation/format_coordinates/FormatCoordinatesSample.java]: throws not needed

**Review by john0005 — APPROVED** [review]
Minor changes. Looks good.

**Comment by tschie** [comment]
@john0005 Weird about the callout not fitting the string... that's probably a bug. I don't see that on Windows.

**Pull request merged**